client/my-sites/customer-home/cards/actions/quick-links/index.jsx

```diff
@@ -43,14 +44,17 @@ export const QuickLinks = ( {
 	trackAddEmailAction,
 	trackAddDomainAction,
 	isExpanded,
-	expand,
-	collapse,
+	updateHomeQuickLinksToggleStatus,
 	isUnifiedNavEnabled,
 	siteAdminUrl,
 	editHomePageUrl,
 	siteSlug,
 } ) => {
 	const translate = useTranslate();
+	const [ debouncedUpdateHomeQuickLinksToggleStatus ] = useDebouncedCallback(
```

Comment:
Oh interesting, I see didn't notice we were using quite an old version of the package! [Docs for version 3.2](https://github.com/xnimorz/use-debounce/blob/3.2.0/README.md) (which is close to the v3.1 we're using). So I didn't realise we'd have to destructure the debounced function like this.

<img width="604" alt="Screenshot 2021-08-04 at 6 10 46 PM" src="https://user-images.githubusercontent.com/1500769/128130768-ff1ffedc-b3a3-42af-aa52-17b377d80457.png">


It looks like the version we're using also doesn't document the `.flush()` function and instead the hook returns a `callPending` function.

```suggestion
	const [ debouncedUpdateHomeQuickLinksToggleStatus, _, callPending ] = useDebouncedCallback(
```

Reply:
Oops...I don't notice that the doc version 😅